nsiqcppstyle_exe.py

```diff
@@ -72,6 +72,10 @@ def Usage():
                 (target/filefilter.txt)
                 If you provide the file path (not a folder path) for the target,
                 -f option should be provided.
+  --filter-string=<filter string>
```

Comment:
This should be exclusive with `-f`

Reply:
Why does it need to be exclusive?  Why not leave it up to the user to find a reason for wanting to use both (e.g., adding a single rule to an already existing filefilter.txt)?

Reply:
Because of no details on what overrides what, and how.

It's easier to remove the restriction later rather than try to keep stability of cli with a bad initialization

Reply:
Sounds logical, code now checks for mutual exclusivity.

Done.